app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -309,7 +315,14 @@ function ConfirmStep({
               <Icon as={MdOutlinePeopleAlt} boxSize={6} mt={1} mr={2} />
               <Box>
                 <Text fontSize="xl">
-                  3,978.9M
+                  {population ? (
+                    <>
+                      {shortenNumber(population)}
+                      {getShortenNumberUnit(population)}
```

Comment:
Nit: I would prefer `getShortenedNumberUnit` for the method name, or making it one method (that the `shortenNumber` method returns a formatted string instead of just the number part).

Reply:
I see that you're using both methods separately further down, so never mind that second suggestion :sweat_smile: